common/spring-cloud-stream-modules-common-configuration/src/main/java/org/springframework/cloud/stream/module/sftp/SftpSessionFactoryConfiguration.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.sftp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.sftp.session.DefaultSftpSessionFactory;
+
+/**
+ * @author Gary Russell
+ */
+@Configuration
+@EnableConfigurationProperties(SftpSessionFactoryProperties.class)
+public class SftpSessionFactoryConfiguration {
+	public static final String SFTP_SESSION_FACTORY_BEAN_NAME = "sftpSessionFactory";
```

Comment:
This (public) does not seem to be referenced from elsewhere. Is it really necessary?